composer.json

```diff
@@ -1,30 +1,29 @@
 {
   "license": "GPL-3.0-or-later",
   "require": {
-    "php": "^7.2",
+    "php": "^7.4",
     "ext-json": "*",
-    "oyejorge/less.php": "~1.5",
+    "ext-intl": "*",
     "mediawiki/oauthclient": "~1.0",
     "wikimedia/slimapp": "dev-master",
     "addwiki/mediawiki-api-base": "~2.1",
     "wikimedia/simplei18n": "~1.0",
     "tedivm/stash": "^0.17",
-    "phpxmlrpc/phpxmlrpc": "^4"
+    "phpxmlrpc/phpxmlrpc": "^4",
+    "wikimedia/less.php": "^4.0"
   },
   "autoload": {
     "psr-4": {
       "Plagiabot\\Web\\": "src/"
     }
   },
   "require-dev": {
-    "jakub-onderka/php-parallel-lint": "^1.0.0",
```

Comment:
The package is still supported and works. The old _name_ was abandoned, is all, and it's now called `php-parallel-lint/php-parallel-lint`.